### PR TITLE
Recipe for MotifBinner2 version 2.0.0

### PR DESCRIPTION
**Pull request by davebx**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).

### CODE REVIEW

**Comment by bgruening** [comment]
```
ERROR: recipes/r-motifbinner/meta.yaml:19: should_be_noarch_generic: The recipe should be build as `noarch`
WARNING: recipes/r-motifbinner/meta.yaml:70: gpl_requires_license_distributed: The recipe packages GPL software but is missing copy of license.
WARNING: recipes/r-motifbinner/meta.yaml:70: long_summary: The summary line is rather long
```

File: recipes/r-motifbinner/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    - r-domc
+    - r-ggplot2
+    - r-knitr
+    - r-optparse
+    - r-rmarkdown
+    - r-foreach
+    - r-tidyr
+    - r-gridextra
+    - r-reshape2
+    - r-stringi
```
Comment (bgruening) [inline recipes/r-motifbinner/meta.yaml]: These deps here do not match the one in the run section. Is really everything needed? And what about r-stringi which is only listed here?

**Comment by dpryan79** [comment]
+1 on merging from me.

**Comment by dpryan79** [comment]
@bioconda-bot merge

**Pull request merged**